package.json

```diff
@@ -20,9 +20,11 @@
         "@babel/register": "^7.12.13",
         "@tsconfig/node14": "^1.0.1",
         "@types/jasmine": "3.6.2",
+        "@wdio/appium-service": "^6.12.1",
```

Comment:
do we need to actually start the appium server ourselves when it's the purpose of this actual package?

Reply:
for using local test device, yes we need a appium server locally.

Reply:
I mean do you really need to start the server locally? I though this service is spawning an instance of the appium server automatically

Reply:
yes, we need this to launch an appium server via this service in local when running test. Otherwise, need to manually launch one before test.